gammapy/astro/population/simulate.py

```diff
@@ -384,14 +384,15 @@ def add_observed_parameters(table, obs_pos=None):
     Parameters
     ----------
     table : `~astropy.table.Table`
-        Input table
+        Input table.
     obs_pos : tuple or None
```

Comment:
Here, should we put `tuple, optional` and then in the description explain that if `None`, default to Earth position ?

Reply:
Thanks @MRegeard 
That's a nice idea